apps/frontend/src/libs/helpers/avatar-helper.ts

```diff
@@ -0,0 +1,11 @@
+const ZERO = 0;
+
+function getAvatarInitials(name: null | string | undefined): string {
```

Comment:
we shouldn't check whether name is exist in this function, and we don't need to check if it's empty. Let's pass the user to the header component from the main page and render header only if user exists. This header will be shown on pages only when the user is present, also we will trim the name during sign-up. So the only action you need to do is to name[0], and I think we don't need a separate helper for this.